k9mail/src/main/java/com/fsck/k9/view/MessageCryptoDisplayStatus.java

```diff
@@ -237,12 +252,15 @@ private static MessageCryptoDisplayStatus getDisplayStatusForPgpResult(CryptoRes
             }
         }
 
+        final long delta = signatureResult.getSignatureTimestamp().getTime() - messageSentDate.getTime();
```

Comment:
`MessageCryptoDisplayStatus` is a dumb translation of the status we get from the crypto provider into something that can be displayed in the UI, it shouldn't make actual decisions of its own about this state.

That said, I'm not sure where this logic actually belongs. Almost all of these decisions are left up to OpenKeychain, I wonder if we should also keep this over there? If not it should probably be its own, testable decider class. Not sure @cketti @dschuermann